tests/unit/test_report_builder.py

```diff
@@ -28,7 +28,7 @@ def before_listener():
     before_fired = True
```

Comment:
Exclude the output results from the repository pls. 
I.e. *.pdf, *.aux, *.synctex.gz ...